src/main/java/com/vmware/vipclient/i18n/messages/service/StringService.java

```diff
@@ -36,19 +36,21 @@ public String getString(MessagesDTO dto) {
     	Iterator<Locale> fallbackLocalesIter = LocaleUtility.getFallbackLocales().iterator();
```

Comment:
This is only used in the if block below.

Reply:
It is used twice: in lines 41 and 42.